tests/test-cases/Group1-Docker-Commands/1-03-Docker-Images.robot

```diff
@@ -30,19 +30,19 @@ Simple images
     ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} pull alpine:3.1
     Should Be Equal As Integers  ${rc}  0
     Should Not Contain  ${output}  Error
-    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} images
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} images alpine
```

Comment:
Is this necessary because the shared VCH will have more images? Would this shared VCH have `alpine` more than 3 times?

Reply:
yes, because we have one vch and there are other images.